src/node_contextify.cc

```diff
@@ -935,6 +935,23 @@ Maybe<bool> StoreCodeCacheResult(
   return Just(true);
 }
 
+// TODO(RaisinTen): Reuse in ContextifyContext::CompileFunction().
+MaybeLocal<Function> CompileFunction(Isolate* isolate,
```

Comment:
This can just accept the context, the isolate can be retrieved via `context->GetIsolate()`